src/connectors/archive.js

```diff
@@ -34,7 +34,7 @@ function bindNew() {
 
 	Connector.getArtistTrack = () => {
 		let track = $('.jwrowV2.playing .ttl').text();
-		const artist = $('.info-top .metadata-definition > dd').first().text();
+		const artist = $('.key-val-big a span').first().text();
```

Comment:
```suggestion
		const artist = $('.key-val-big a span, .metadata-definition > dd').first().text();
```

Since this breaks other pages such as: https://archive.org/details/AH013_sarin_sunday_-_the_lonely_hike/01_Light_Fragments.mp3